types/kendo-ui/index.d.ts

```diff
@@ -3748,7 +3748,7 @@ declare namespace kendo.ui {
         aggregates?: any;
         attributes?: any;
         columns?: any;
-        command?: GridColumnCommandItem[];
+        command?: (string|GridColumnCommandItem)[];
```

Comment:
Are you sure you don't want `string[] | GridColumnCommandItem[]` here? Can the array be a mixture of `string`s and `GridColumnCommandItem`s?